controllers/rhmi/rhmi_controller.go

```diff
@@ -688,6 +689,20 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 		}
 	}
 
+	if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi) {
+		ok, err := addon.ExistsParameterByInstallation(context.TODO(), r.Client, installation, addon.QuotaParamName)
+		if err != nil || !ok {
+			var preflightMessage string
+			if err != nil {
+				preflightMessage = fmt.Sprintf("failed to retrieve addon parameter %s: %v", addon.QuotaParamName, err)
+			} else {
+				preflightMessage = fmt.Sprintf("parameter %s could not be found", addon.QuotaParamName)
+			}
+
+			log.Info(preflightMessage)
```

Comment:
Should there be a return here ? Preflght checks looks to progress and succeed even if the parameter is not found :thinking:

Reply:
@KevFan  It should. Missed that after reverting the initial refactor :stuck_out_tongue: